File: responses/Z9S2T8-8XQ/sum_of_integers_in_a_string.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(S) {
    // write your code in JavaScript
    
    var splitArray = S.split(",")
    
    var sum = 0
    
    for(i = 0; i < splitArray.length; i++) {
        sum = sum + parseInt(splitArray[i])
    }
    
    return sum
}

File: responses/N7WCXZ-Y9Y/sum_of_integers_in_a_string.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(S):
    # write your code in Python
    num = list(map(int,S.split(',')))
    
    sum=0
    for it in num:
        sum += int(it)
        
    return sum
    